shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -401,14 +402,16 @@ public void writeFields(DataOutput out) throws IOException {
             out.writeLong(requestId);
             out.writeLong(writerId.getMostSignificantBits());
             out.writeLong(writerId.getLeastSignificantBits());
+            out.writeUTF(delegationToken == null ? "" : delegationToken);
```

Comment:
This isn't backwards compatible. You are writing 2 UTF Strings here and reading 2 below. If we have a version mismatch then below you will read the Segment name as your delegation token and have an empty/undefined Segment name.

Reply:
Yes. We need to bump up the protocol version.